### DISCUSSION TITLE
Building for mac

### DISCUSSION THREAD

**Discussion by vernoceedee** [Snapmaker/Luban#1672] [Q&A]
is there a build walk-through for building this on Mac? Sorry I'm a bit lost.

**Comment by vernoceedee** [comment]
guess not :(

**Comment by tgunr** [comment]
I can get it to run using `rpm run dev` but don't see a way to build the app. Trying `nom run prod` fails.

**Reply by parachvte** [reply]
There are two steps to build the application:

1. Run `npm run build` to build
2. Run `npm run build:mac-x64` to build installer (if you are building on macOS 64-bit, other systems using their corresponding command)

**Reply by tgunr** [reply]
Yea! That worked, will add a note to the docs.
